.github/workflows/langchain_ci.yml

```diff
@@ -33,77 +33,29 @@ env:
 
 jobs:
   lint:
-    uses:
-      ./.github/workflows/_lint.yml
+    uses: ./.github/workflows/_lint.yml
     with:
       working-directory: libs/langchain
-      langchain-core-location: ../core
     secrets: inherit
 
   test:
-    uses:
-      ./.github/workflows/_test.yml
+    uses: ./.github/workflows/_test.yml
     with:
       working-directory: libs/langchain
-      langchain-core-location: ../core
     secrets: inherit
 
   compile-integration-tests:
-    uses:
-      ./.github/workflows/_compile_integration_test.yml
+    uses: ./.github/workflows/_compile_integration_test.yml
     with:
       working-directory: libs/langchain
-      langchain-core-location: ../core
     secrets: inherit
 
   pydantic-compatibility:
-    uses:
-      ./.github/workflows/_pydantic_compatibility.yml
+    uses: ./.github/workflows/_pydantic_compatibility.yml
     with:
       working-directory: libs/langchain
-      langchain-core-location: ../core
     secrets: inherit
 
-  # It's possible that langchain works fine with the latest *published* langchain-core,
```

Comment:
Note this test was actually already duplicate, because when the main test workflow is run with `langchain-core-location` arg it always installs the in-tree version of core